src/components/Description/Description.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import Description from "./Description";
+
+import "./../../common.css";
+
+export default {
+  component: Description,
+  title: "Components/Description",
+};
+
+export const Primary = () => <Description />;
```

Comment:
Add the example text in this file and pass it here to the `Description` component. You can see how I did it in my timeline component.